robotbase/src/main/java/com/team2357/frc2022/subsystems/FeederSubsystem.java

```diff
@@ -28,10 +28,7 @@ public void runFeedermotor(double speed) {
     // flip that to true for reability
     public boolean isBallAtFeederWheel() {
         boolean isSensorBlocked = false;
-        if (m_arduinoIRSensor.isConnected()) {
-            isSensorBlocked = !m_arduinoIRSensor.getDeviceFieldBoolean(Constants.ARDUINO.IR_SENSOR_JSON_NAME,
-                    "state");
-        }
+        isSensorBlocked = m_outputIRSensor.getFeederValue();
```

Comment:
This method will also go away eventually, but could be simplified by just returning m_outputIRSensor.getFeederValue().